### ISSUE TITLE
pypi distribution is not working perfectly

### ISSUE THREAD

**Issue by shimwell** [makeclean/parametric-plasma-source#11]
the install option of ```pip install parametric-plasma-source``` is not quite working

I think this is because the newest version does some cmake commands

One solution might be to move to a conda install / conda distribution

![Screenshot from 2020-09-22 15-22-54](https://user-images.githubusercontent.com/8583900/93895480-c6bad780-fce7-11ea-9aa7-63b72b066ccb.png)

**Comment by shimwell** [comment]
In the mean time users can always clone the repo and install using ```python setup.py install```

**Comment by shimwell** [comment]
Thanks for taking a look at this @DanShort12 and pointing out that the pypi upload doesn't contain the wheels.

**Comment by shimwell** [comment]
The python.yml github actions has an upload wheels stage but I'm not sure if this line does anything at the moment https://github.com/makeclean/parametric-plasma-source/blob/e28622d4c34d905fe07f73b38e5ebda786b9caab/.github/workflows/python.yml#L42

**Comment by DanShort12** [comment]
> The python.yml github actions has an upload wheels stage but I'm not sure if this line does anything at the moment
> 
> https://github.com/makeclean/parametric-plasma-source/blob/e28622d4c34d905fe07f73b38e5ebda786b9caab/.github/workflows/python.yml#L42

The 'upload-artifact' action will make the artifacts available outside of the action e.g. [this](https://github.com/makeclean/parametric-plasma-source/suites/1208149264/artifacts/17940721), but won't 'upload' in the sense of putting it on PyPI.

**Comment by DanShort12** [comment]
I had a go at the PyPI upload on TestPyPI and realized that this isn't building a [manylinux](https://github.com/pypa/manylinux) package. I might have to take a look at getting the relevant docker image (or to work out why the install from tar.gz can't find cmake!).

**Comment by qingfengxia** [comment]
The cmake assuming openmc is installed to `/opt/openmc`  while the installation can be detected by cmake: find_program() just as
`which openmc` will give the full path of openmc,  from that openmc_DIR can be retrieved.

**Comment by DanShort12** [comment]
Fair point that the OpenMC install location isn't being detected (that's a hand over from the original makefile implementation). However, I think the issue @Shimwell reports here is related to pip not using the wheel that's created by CI, due to the install not using a manylinux-compatible image.
